packages/messaging/src/controllers/sw-controller.test.ts

```diff
@@ -218,6 +218,30 @@ describe('SwController', () => {
       expect(postMessageSpy).to.have.been.calledOnceWith(expectedMessage);
     });
 
+    it('sends a message to chrome extension clients if a chrome extension client is visible', async () => {
+      const client: Writable<WindowClient> = (await self.clients.openWindow(
+        'chrome-extension://aihpiglmnhnhijdnjghpfnlledckkhja/popup.html'
```

Comment:
looks like a real extension id. can we have it to be more generic? something like "chrome-extension://testExtensionId/popup.html" or whatever.